### PR TITLE
docs: update life_of_a_query.md

### PR DESCRIPTION
**Pull request by louishust**

### CODE REVIEW

File: docs/tech-notes/life_of_a_query.md [hunk 1]
```diff
@@ -965,10 +965,10 @@ sure they're not writing "under" a read that has already been
 performed.
 
  Now we're reading to actually evaluate the read - control moves to
- [`replica.executeBatch()`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica.go#L4000)
+ [`replica.evaluateBatch()`](https://github.com/cockroachdb/cockroach/blob/7bed6f56e63355d53b2d28c3fc623ab9adac099f/pkg/storage/replica.go#L5225)
  which calls
- [`replica.executeCmd`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica.go#L4050)
- for each request in the batch. `executeCmd` switches over the request
+ [`replica.evaluateCommand`](https://github.com/cockroachdb/cockroach/blob/7bed6f56e63355d53b2d28c3fc623ab9adac099f/pkg/storage/replica.go#L67)
```
Comment (couchand) [inline docs/tech-notes/life_of_a_query.md]: I think this link is pointing to the wrong place?

File: docs/tech-notes/life_of_a_query.md [hunk 2]
```diff
@@ -1129,12 +1129,12 @@ method. This takes in the `roachpb.BatchRequest` (the KV request we've
...
 `engine.Batch`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica.go#L3835)
 and delegates to
-[`executeBatch()`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica.go#L3850). This
+[`evaluateBatch()`](https://github.com/cockroachdb/cockroach/blob/7bed6f56e63355d53b2d28c3fc623ab9adac099f/pkg/storage/replica.go#L5225). This
 fellow finally
 [iterates](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica.go#L4041)
 over the individual requests in the batch and, for each one, calls
-[`executeCmd`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica_command.go#L115). We've
-seen `executeCmd` before, on the read path. It switches over the
+[`evaluateCommand`](https://github.com/cockroachdb/cockroach/blob/33c18ad1bcdb37ed6ed428b7527148977a8c566a/pkg/storage/replica_command.go#L67). We've
```
Comment (couchand) [inline docs/tech-notes/life_of_a_query.md]: Ditto.

**Review by couchand — COMMENTED** [review]
Thanks for looking at updating this!

**Comment by louishust** [comment]
@couchand  I've modified the link

**Review by couchand — APPROVED** [review]
Looks good, thanks!

**Comment by louishust** [comment]
@couchand  all check passed

**Pull request merged**